static/app/views/alerts/rules/issue/index.tsx

```diff
@@ -942,6 +950,53 @@ class IssueRuleEditor extends DeprecatedAsyncView<Props, State> {
     );
   }
 
+  displayNoConditionsWarning(): boolean {
+    const {rule} = this.state;
+    return (
+      this.props.organization.features.includes('noisy-alert-warning') &&
+      !!rule &&
+      !isSavedAlertRule(rule) &&
+      rule.conditions.length === 0
```

Comment:
What about filters? On the back end we combine filters and conditions to just be conditions but I think here they're separate. I think we're ok with no conditions as long as there are filters.